itou/fixtures/django/04_test_users.json

```diff
@@ -31,7 +31,7 @@
       "last_login": "2021-10-07T15:28:36.090Z",
       "last_name": "",
       "nir": "",
-      "password": "pbkdf2_sha256$260000$7SMYXy9OTGSwYOs5myl9bP$GVwB+ayk8qAxkWOmP62sO+qGm1qyzivafdADRDip2/w=",
+      "password": "pbkdf2_sha256$390000$eNyrLRNGvCTRIfxDApp5nA$zMJxqzLJzlu20MhT85T/9UQJ2M/Tq/rNTrBS56a+tX8=",
```

Comment:
comment se fait-il que les mots de passe changent (ou du moins leur hash ?) pure curiosité.

Reply:
Honnêtement, je ne sais pas.

Reply:
C’est probablement à cause d’une montée de version de Django depuis que les fixtures ont été générées.
https://docs.djangoproject.com/en/4.2/releases/4.1/#django-contrib-auth

En général, Django augmente les itérations au fur et à mesure des releases pour rester à environ 1s de temps de calcul sur un processeur moderne.

Je ne sais pas comment tu les as mises à jour, mais à un moment, il y a sûrement eu l’occasion de hasher à nouveau, et django a mis à jour le hash avec le nombre d’itérations récent.